.github/workflows/build_navitia_packages_for_release.yml

```diff
@@ -22,25 +27,26 @@ jobs:
         sed -i 's,git\@github.com:\([^/]*\)/\(.*\).git,https://github.com/\1/\2,' .gitmodules
         git submodule update --init --recursive
         DEB_BUILD_OPTIONS=nocheck dpkg-buildpackage -b
-    - name: create navitia_debian_packages.zip
+    - name: create navitia_${{matrix.distribution}}_packages.zip
       run: |
-        zip navitia_debian_packages.zip ../navitia-*
-    - name: upload debian packages
+        zip navitia_${{matrix.distribution}}_packages.zip ../navitia-*
+    - name: upload ${{matrix.distribution}} packages
       uses: actions/upload-artifact@v1
       with:
-        name: navitia-debian-packages
-        path: "navitia_debian_packages.zip"
+        name: navitia-${{matrix.distribution}}-packages
+        path: "navitia_${{matrix.distribution}}_packages.zip"
     - name: remove useless temporary files
       run: rm -rf ../navitia-*
-    - name: trigger jenkins to publish Artifacts on internal server
-      run: http -v -f POST https://${{secrets.jenkins_token}}@ci.navitia.io/job/publish_navitia_release_packages_from_github/build
+
     - name: slack notification (the job has failed)
       if: failure()
       run: |
           echo '{"text":":warning: Github Actions: build_navitia_packages_for_release failed ! (https://github.com/CanalTP/navitia/actions?query=workflow%3A%22Build+Navitia+Packages+For+Release%22). Navboys, this is a release alert !!!"}' | http --json POST ${{secrets.SLACK_NAVITIA_CORE_TEAM_URL}}
-    - name: slack notification (the job has successed)
-      if: success()
+
+    - name: trig publishing + notifications (the job has successed)
+      if: success() && matrix.distribution == 'debian8'
       run: |
+          http --ignore-stdin -v -f POST https://${{secrets.JENKINS_NG_TOKEN}}@jenkins-core.canaltp.fr/job/publish-navitia-packages/buildWithParameters PLATFORM=navitia_debian8_packages
```

Comment:
Trig the **publishing job** (https://jenkins-core.canaltp.fr/job/publish-navitia-packages/)
This job work with **multi-packages** artifacts